client/my-sites/domains/domain-management/gsuite/gsuite-add-users/add-users.jsx

```diff
@@ -31,6 +31,11 @@ import { filter as filterUsers, validate as validateUsers } from 'lib/domains/go
 import DomainsSelect from './domains-select';
 import QueryUserSettings from 'components/data/query-user-settings';
 
+/**
+ * Style dependencies
+ */
+import './add-users.scss';
+
 function getGoogleAppsCartItems( { domains, fieldsets } ) {
```

Comment:
Should we rename this function now, or do all renamings in another pull request?